README.md

```diff
@@ -222,6 +222,11 @@ results" in the Publish JUnit test result report configuration.
 
 ## Change Log
 
+### Version 0.71 (21 Sep 2021)
+- Updated scripts to use a different method for checking the presence of the
+  new reports api. Later versions of VectorCAST will require this version of the
```

Comment:
This should just say “DataAPI”.

Reply:
@nbelliot Do you agree on this?

Reply:
"The Python support scripts have been updated to attempt importing a module from VectorCAST's DataAPI/reporting API vs. checking for the existence of a pre-compiled Python bytecode file. Such an approach follow's Python's [\"EAFP\"](https://docs.python.org/3.5/glossary.html#term-eafp) mantra."

(and then don't mention the bit about "later versions" -- a version _today_ and a version after 92851 will both be able to import that module vs. checking for the pyc).

Reply:
@vapdrs Could you please update the release notes as @andrewvaughanj suggested? Thanks!